### PR TITLE
new jdk8 container with coverity

### PR DESCRIPTION
**Pull request by elduddz**

### CODE REVIEW

File: linux/jdk8-python36-cov/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  && pip3 install setuptools==41.6.0 wheel==0.33.6 \
+  && pip3 install awscli==1.16.277 \
+  && curl -L -O https://amazon-ecr-credential-helper-releases.s3.us-east-2.amazonaws.com/0.4.0/linux-amd64/docker-credential-ecr-login \
+  && chmod +x ./docker-credential-ecr-login \
+  && mv ./docker-credential-ecr-login /usr/local/bin/
+COPY ./cov-analysis-linux64-2019.12.tar.gz /tmp/cov-analysis-linux64-2019.12.tar.gz
+RUN tar -xvf "/tmp/cov-analysis-linux64-2019.12.tar.gz" -C /usr/local/bin
+ENV COVERITY_TOOL_HOME=/usr/local/bin/cov-analysis-linux64-2019.12
+ENV PATH=${PATH}:${COVERITY_TOOL_HOME}/bin
+RUN rm /tmp/*
```
Comment (eganjs) [inline linux/jdk8-python36-cov/Dockerfile:28]: Add a new line at the end of the file

Comment (eganjs) [inline linux/jdk8-python36-cov/Dockerfile:28]: Perhaps
```suggestion
RUN rm -rf /tmp/*
```

Comment (elduddz) [reply linux/jdk8-python36-cov/Dockerfile:28]: I have tested it local and I think the rm does remove the content, I will rf it for saftey, the problem i think I had was I broke the PATH on a run, so I couldnt get rm to work at all ;)

Comment (elduddz) [reply linux/jdk8-python36-cov/Dockerfile]: done

**Pull request merged**